File: OLSource1/v1/VS_officedev/codesnippet/CSharp/how-to--remove-managed-code-extensions-from-documents_2.cs
            string documentPath = System.Environment.GetFolderPath(
                Environment.SpecialFolder.Desktop) + @"\WordDocument1.docx";
            int runtimeVersion = 0;

            try
            {
                runtimeVersion = ServerDocument.GetCustomizationVersion(documentPath);

                if (runtimeVersion == 3)
                {
                    ServerDocument.RemoveCustomization(documentPath);
                    System.Windows.Forms.MessageBox.Show("The customization has been removed.");
                }
            }
            catch (FileNotFoundException)
            {
                System.Windows.Forms.MessageBox.Show("The specified document does not exist.");
            }
            catch (IOException)
            {
                System.Windows.Forms.MessageBox.Show("The specified document is read-only.");
            }
            catch (InvalidOperationException ex)
            {
                System.Windows.Forms.MessageBox.Show("The customization could not be removed.\n" +
                    ex.Message);
            }

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/how-to--remove-managed-code-extensions-from-documents_2.vb
        Dim documentPath As String = System.Environment.GetFolderPath( _
            Environment.SpecialFolder.Desktop) & "\WordDocument1.docx"
        Dim runtimeVersion As Integer = 0

        Try
            runtimeVersion = ServerDocument.GetCustomizationVersion(documentPath)
            If runtimeVersion = 3 Then
                ServerDocument.RemoveCustomization(documentPath)
                System.Windows.Forms.MessageBox.Show("The customization has been removed.")
            End If
        Catch ex As FileNotFoundException
            System.Windows.Forms.MessageBox.Show("The specified document does not exist.")
        Catch ex As IOException
            System.Windows.Forms.MessageBox.Show("The specified document is read-only.")
        Catch ex As InvalidOperationException
            System.Windows.Forms.MessageBox.Show("The customization could not be removed." & _
                vbLf & ex.Message)
        End Try